lblod/app-lokaal-mandatenbeheer
# Lmb 1111 | improve linked mandatarissen migrations

## Description

Migrations do not always run. Improve the queries so they are less heavy.

## How to test

 - Start from prod db dump
- Let the migrations run and see if you have any hickups while migrating 

## Links to other PR's

- https://github.com/lblod/app-lokaal-mandatenbeheer/pull/316
